oauthlib/oauth2/rfc6749/endpoints/pre_configured.py

```diff
@@ -49,33 +52,37 @@ def __init__(self, request_validator, token_expires_in=None,
         refresh_grant = RefreshTokenGrant(request_validator)
         openid_connect_auth = OpenIDConnectAuthCode(request_validator)
         openid_connect_implicit = OpenIDConnectImplicit(request_validator)
+        openid_connect_hybrid = OpenIDConnectHybrid(request_validator)
 
         bearer = BearerToken(request_validator, token_generator,
                              token_expires_in, refresh_token_generator)
 
-        auth_grant_choice = AuthCodeGrantDispatcher( default_auth_grant=auth_grant, oidc_auth_grant=openid_connect_auth)
+        auth_grant_choice = AuthCodeGrantDispatcher(default_auth_grant=auth_grant, oidc_auth_grant=openid_connect_auth)
+        implicit_grant_choice = ImplicitTokenGrantDispatcher(default_implicit_grant=implicit_grant, oidc_implicit_grant=openid_connect_implicit)
 
         # See http://openid.net/specs/oauth-v2-multiple-response-types-1_0.html#Combinations for valid combinations
         # internally our AuthorizationEndpoint will ensure they can appear in any order for any valid combination
         AuthorizationEndpoint.__init__(self, default_response_type='code',
                                        response_types={
                                            'code': auth_grant_choice,
-                                           'token': implicit_grant,
-                                           'id_token': openid_connect_implicit,
-                                           'id_token token': openid_connect_implicit,
-                                           'code token': openid_connect_auth,
-                                           'code id_token': openid_connect_auth,
-                                           'code token id_token': openid_connect_auth,
+                                           'token': implicit_grant_choice,
+                                           'id_token': implicit_grant_choice,
```

Comment:
Should `id_token` and `id_token token` not point to the `openid_connect_implicit` grant directly?